docs/integration_tests.md

```diff
@@ -145,125 +123,69 @@ In case you plan to use Minishift, the following steps are necessary to get Kept
    ```console
-   keptn configure bridge --action=expose
+   curl -SL https://raw.githubusercontent.com/keptn/examples/master/quickstart/expose-keptn.sh | bash
    ```
-1. Verify accessing Keptn Bridge works
-1. Configuration for the test:
+5. Open a new terminal and type:
    ```console
-   export PROJECT=easytravel
-   export SERVICE=frontend
-   export STAGE=hardening
-   export QG_INTEGRATION_TEST_DT_TENANT=<INSERT_YOUR_DT_TENANT_HERE>
-   export QG_INTEGRATION_TEST_DT_API_TOKEN=<INSERT_YOUR_DT_API_TOKEN_HERE>
+   kubectl -n keptn port-forward service/api-gateway-nginx 8080:80
    ```
-1. Run the test
+   After executing, return back to the original terminal.
+6. Authenticate Keptn:
    ```console
-   bash ./test/test_quality_gates_standalone.sh
+   keptn auth --endpoint=http://127.0.0.1:8080/api --api-token=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
    ```
-
-### Run the full installation integration test locally
-
-Pre-requisites:
-
-* A K8s cluster (see above)
-  * 12 GB of free memory (less is possible but you will notice a significant slowdown)
-  * at least 6 CPU cores on your system (less is possible but you will notice a significant slowdown)
-* `kubectl`
-* `keptn` CLI
-* A local copy of this repository (keptn/keptn)
-* Linux, Mac OS or Linux Subsystem for Windows
-* Bash
-* working internet connection
-
-1. Setup your Kubernetes cluster (see above)
-1. Verify `kubectl` is configured to the right cluster:
+7. Verify the installation has worked
    ```console
-   kubectl get nodes
+   keptn status
    ```
-1. `keptn` CLI available
-1. Install Keptn
-   * K3s
-       ```console
-       keptn install --use-case=continuous-delivery --verbose
-       ```
-   * Minikube
-       ```console
-       keptn install --use-case=continuous-delivery --endpoint-service-type=NodePort --verbose
-       ```
-       **Note**: You can use `--endpoint-service-type=LoadBalancer` if you run `minikube tunnel`
-   * Minishift
-      ```console
-      keptn install --use-case=continuous-delivery --platform=openshift --verbose
-      ```
-   **Note**: Use `--chart-repo=https://storage.googleapis.com/keptn-installer/latest/keptn-0.1.0.tgz` to install the latest master branch, see [master installation docs](install_master.md).
-1. Verify the installation has worked
+8. Verify which images have been deployed
    ```console
-   keptn status
+   kubectl -n keptn get deployments
    ```
-1. Verify which images have been deployed
+9.  Run tests (e.g., UniformRegistration):
    ```console
-   kubectl -n keptn get deployments -owide
+   cd test/go-tests && KEPTN_ENDPOINT="http://127.0.0.1:8080/api" go test ./...
    ```
-1. Run tests (e.g., UniformRegistration):
+   **Note**: If you want to run a single test, (e.g. BackupTestore_Test), please add `_test` suffix to the test file name, so it becomes executable. Otherwise, you will be able to run only the `testsuite_*_test.go` files. For running a single test use:
```

Comment:
```suggestion
   **Note**: If you want to run a single test, (e.g. BackupTestore_Test), please add `_test` suffix to the test file name, so it becomes executable. Otherwise, you can run only `testsuite_*_test.go` files. For running a single test use:
```